src/mmw/js/src/analyze/views.js

```diff
@@ -196,11 +199,14 @@ var TabContentView = Marionette.LayoutView.extend({
         chartRegion: '.analyze-chart-region'
     },
     onShow: function() {
-        var dataCollection = new coreModels.DataCollection(
-            this.model.get('categories')
-        );
+        var categories = this.model.get('categories'),
+            largest = _.max(_.map(categories, function(cat) {return cat.area;})),
```

Comment:
This might be more easily written as:

```javascript
largest = _.max(categories.pluck('area'));
```

See [pluck](http://backbonejs.org/#Collection-pluck)